src/app/cli/src/commands/sql_shell_command.rs

```diff
@@ -124,17 +128,30 @@ impl SqlShellCommand {
 
         Ok(())
     }
+
+    async fn run_datafusion_cli_command(&self) -> Result<(), CLIError> {
+        let mut print_options = PrintOptions {
+            format: PrintFormat::Table,
+            quiet: true,
+            maxrows: MaxRows::Unlimited,
```

Comment:
Perhaps for REPL it's not a good idea to have unlimited output rows - it's so easy to forget to add `LIMIT` and end up spamming your shell with massive amount of data.